python/shipDigiReco.py

```
@@ -75,43 +83,93 @@ def __init__(self, finput, fout, fgeo) -> None:
+
     def reconstruct(self) -> None:
-        self.findTracks()
-        self.findGoodTracks()
+        # self.findTracks()
+        self.actsTracks()
         if hasattr(self, "digiSBT"):
-            self.linkVetoOnTracks()
-        if global_variables.vertexing:
-            # now go for 2-track combinations
-            self.Vertexing.execute()
+            calculate_sbt_doca(self, output_tracks, self.digiSBT.det)
```

Ruff (0.15.12)

[error] Undefined name `output_tracks`

(F821)